sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar-project.properties
+sonar.organization=fedora-infra
+sonar.projectKey=fedora-infra/noggin
+sonar.sources=.
+sonar.host.url=https://sonarcloud.io
+sonar.login=a36fe78f64abe3b8e6db0df06a6b399c2abc2e24
+sonar.test.exclusions=**/noggin/tests/*
+sonar.exclusions=/devel/create-test-data.py, /noggin/themes/*, /noggin/templates/*, 
```

Comment:
Not sure that paths should begin with a `/`. Also, why should we exclude those paths?

Reply:
The create-test-data file is just a way for us to create users and groups so that there is something in "noggin", not something we need sonarqube to test against. RLerch let me know that the other two paths, "themes" and "templates" are where the bundled libraries are kept and also shouldn't be tested against.

Reply:
The themes also contain templates and I thought we wanted to test them, for example that's where the fixes on the `alt` tag for images came from. If we want to keep templates but drop bundled libraries, maybe we could use something like `noggin/themes/*/static/`?

Reply:
that has been changed